### PR TITLE
App intro page landing screen added

### PR DESCRIPTION
**Pull request by Aswinhub**
App intro screen UI created as per the video.
A landing app intro activity which show two fragments separately.
For portrait and landscape mode two xml created with corresponding orientations with different fragment alignments.
Fragment action will show the user options like sign in or signup
Fragment view will display a view pager with multiple images.
Drawable file are created for circular image and button background

### CODE REVIEW

File: app/src/main/java/com/aswin/testapp/fragments/FragmentView.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_view, container, false);
+        initView(view);
+        return view;
+    }
+
+    private void initView(View view) {
+        viewPager = (ViewPager) view.findViewById(R.id.viewPager);
+        pagerIndicator = (LinearLayout) view.findViewById(R.id.pager_indicator);
+
```
Comment (Ankish) [inline app/src/main/java/com/aswin/testapp/fragments/FragmentView.java]: avoid extra space

File: app/src/main/java/com/aswin/testapp/fragments/FragmentView.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        pagerAdapter = new ViewPagerAdapter(getActivity(), 5);
+        viewPager.setAdapter(pagerAdapter);
+        viewPager.setCurrentItem(0);
+        viewPager.addOnPageChangeListener(this);
+        createIndicator();
+    }
+
+    private void createIndicator() {
+        int count = pagerAdapter.getCount();
+
```
Comment (Ankish) [inline app/src/main/java/com/aswin/testapp/fragments/FragmentView.java]: avoid extra spaces

File: app/src/main/java/com/aswin/testapp/fragments/FragmentView.java (hunk 1 above)
Comment (Aswinhub) [reply app/src/main/java/com/aswin/testapp/fragments/FragmentView.java]: spacing removed

File: app/src/main/java/com/aswin/testapp/fragments/FragmentView.java (hunk 2 above)
Comment (Aswinhub) [reply app/src/main/java/com/aswin/testapp/fragments/FragmentView.java]: spacing removed

**Comment by Ankish** [comment]
@Aswinhub : Can you please rebase and squash

**Pull request merged**